python/ray/data/_internal/execution/streaming_executor.py

```diff
@@ -107,10 +107,20 @@ def execute(
         self._start_time = time.perf_counter()
 
         if not isinstance(dag, InputDataBuffer):
-            logger.get_logger().info("Executing DAG %s", dag)
-            logger.get_logger().info("Execution config: %s", self._options)
+            stdout_logger = logger.get_logger()
+            log_path = logger.get_datasets_log_path()
+            message = (
+                "Starting execution of Dataset. Monitor progress on Ray "
+                "Dashboard."
```

Comment:
I feel like the "Monitor progress on Ray Dashboard" part isn't helpful since you already receive a message when Ray initializes instructing you to view the dashboard.

Reply:
I was wanted to emphasize that people should use dashboard for monitoring. I can also remove this, WDYT? @raulchen.

Reply:
Removed for now.